src/components/accession2/withdraw/WithdrawModal.tsx

```diff
@@ -41,12 +44,25 @@ export default function WithdrawDialog(props: WithdrawDialogProps): JSX.Element
     notes: '',
   };
 
+  const nurseryTransfer: NurseryTransfer = {
+    date: getTodaysDateFormatted(),
+    destinationFacilityId: -1,
+    germinatingQuantity: 0,
+    notes: '',
+    notReadyQuantity: 0,
+    readyByDate: undefined,
+    readyQuantity: 0,
+    withdrawnByUserId: user.id,
+  };
+
   const newViabilityTesting: ViabilityTestPostRequest = {
     testType: 'Lab',
     seedsTested: 0,
   };
 
   const [record, setRecord, onChange] = useForm(newWithdrawal);
+  const [nurseryTransferRecord, setNurseryTransferRecord, onChangeNurseryTransfer] = useForm(nurseryTransfer);
```

Comment:
we use 2 types of records because we have 2 separate api - we maintain a copy of the common fields in both records because user can switch purpose and not want to see selection lost in other fields